rclpy/rclpy/node.py

```diff
@@ -31,6 +31,22 @@
 from rclpy.validate_topic_name import validate_topic_name
 
 
+def check_for_type_support(msg_type):
+    try:
+        getattr(msg_type.__class__, '_TYPE_SUPPORT')
```

Comment:
I would suggest to keep the syntax and use the accessed attribute (rather than ignore it):

```
ts = msg_type.__class__._TYPE_SUPPORT
```

Later this can be used:

```
if ts is None:
```

Reply:
How weird I had that before but switched back to `getattr` without assignment for some reason o_O